src/UserFunctions.java

```diff
@@ -121,22 +146,37 @@ private void printCommands() {
         System.out.println(help); 
     }
 
-    private void createNewPuzzle(String seedWord) {
+    private void createNewPuzzle(String seedWord, char requiredLetter) {
+        System.out.println("Generating new puzzle ...");
+
+        String newLine = getNewLineCharacter();
+        try {
+            FileReader dictionaryFile = new FileReader(DICTIONARY_PATH);
+            puzzle = Puzzle.fromWord(seedWord, requiredLetter, dictionaryFile);
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage() + newLine + 
+                               "Puzzle not generated. Please try again.");
+        } catch (FileNotFoundException e) {
+            System.out.println("A dictionary file could not be found." + 
```

Comment:
Maybe this should print out the filepath of the dictionary file that it's looking for? Not a huge deal but it's really helpful when trying to fix a "file not found" problem outside of code.

Reply:
I don't think that I can access what filepath was attempted from the FileNotFoundException object.

Reply:
It would just be the dictionary path constant, I think.